text/0000-bionic-mixins.md

```diff
@@ -0,0 +1,85 @@
+# Meta
+[meta]: #meta
+- Name: Defining Mixins for io.buildpacks.stacks.bionic
+- Start Date: 2019-12-05
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+The Platform API specification allows stack maintainers to define stack mixins for a given stack.
+The Cloud Native Buildpacks project informally defines stack build and run images with ID `io.buildpacks.stacks.bionic` as containing exactly the set of Ubuntu packages in the `ubuntu:bionic` image on Docker Hub.
+This RFC proposes that we formalize the existing definition of `io.buildpacks.stacks.bionic` and define mixins on that stack to be Ubuntu 18.04 LTS packages. 
+
+In general, a stack is a contract that should be more like "Ubuntu Bionic" and less like "Platform XYZ's Re-distribution of Ubuntu Binoic."
+Stack images can still be provided by platforms/vendors, but they should use the standard CNB stack contracts where applicable.
```

Comment:
I find this language a bit confusing.  On first read, I thought it meant that the images *must be derived from* `ubuntu:bionic`, but I think it really means *provides equivalent functionality*  to `ubuntu:bionic`.  Otherwise images like `gcr.io/gcp-runtimes/ubuntu_18_0_4` won't qualify.   And it may be worth expanding _functionality_: does it mean that `/var/lib/dpkg` should be retained and that new packages can be installed with `apt-get`?